pkg/mediatype/types.go

```diff
@@ -0,0 +1,60 @@
+package mediatype
+
+var AAC, _ = NewMediaType("audio/aac", "", "aac")
```

Comment:
Maybe `const` instead of `var` here? Unless it would be useful to allow overwriting the default media types, I'm a bit on the fence for this one.

```suggestion
const AAC, _ = NewMediaType("audio/aac", "", "aac")
```

Reply:
Unfortunately you can't make this const, it's simply not valid go because it's not a constant

Reply:
Alright, `var` it is then